dev.Dockerfile

```
@@ -35,6 +35,7 @@ RUN RUSTFLAGS='-C target-cpu=native' $HOME/.cargo/bin/cargo build --release \
 COPY docker/dev/Settings-dev.toml /app/Settings.toml
 COPY docker/dev/service.sh /etc/service/geoengine/run
 RUN chmod +x /etc/service/geoengine/run
+RUN adduser --disabled-password --gecos "" geoengine
```

```suggestion
RUN chmod +x /etc/service/geoengine/run \
    && \
    adduser --disabled-password --gecos "" geoengine

```

1. Each `RUN` command introduces a new layer in the container, i.e., increases its size.
2. Please give the line that separates the EXPOSE command back for readability :smiley: .